pilosa/orm.py

```diff
@@ -210,13 +210,14 @@ def field(self, name, time_quantum=TimeQuantum.NONE,
         :param int cache_size: Values greater than 0 sets the cache size. Otherwise uses the default cache size
         :param int int_min: Minimum for the integer field
         :param int int_max: Maximum for the integer field
+        :param bool keys: Sets whether field uses string keys
```

Comment:
For consistency, let's change this to `Whether the field uses string keys`

Reply:
Updated